src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The `href` attribute is pointing to an SCSS file (`styles/index.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS file into a CSS file and link to the resulting CSS file instead. For example, if you compile it to `styles/index.css`, update the `href` to `styles/index.css`.